spec/features/pipelines/cherrypick/cherrypick_pipeline_spec.rb

```diff
@@ -190,8 +188,7 @@ def initialize_plates(plates)
         [plates[0], plates[2], plates[1]]
       end
 
-      let(:expected_tecan_file) do
-        <<~TECAN
+      let(:expected_tecan_file) { <<~TECAN }
```

Comment:
has prettier done the right thing there? should the close curly brace not be after the second TECAN? or left as a do end

Reply:
You're right that does look dodgy, thanks